docs/product-installation-guide/src/main/asciidoc/clustering-con.adoc

```diff
@@ -1,15 +1,64 @@
+
+Most development environments consist of {CENTRAL} for creating rules and processes and at least one {KIE_SERVER} to test those rules and processes. 
 
 [NOTE]
 ====
-This section is specifically intended for {CENTRAL} development environments where you want to cluster the Git repository. It is not necessary to create a clustered environment for {KIE_SERVER} production environments.
+You can also use this configuration for runtime environemtns if you want to use {CENTRAL} to manage the {KIE_SERVER}s in the cluster.
 ====
 
-Red Hat recommends that you consider clustering two or more computers to create a high availability clustered environment for design-time development environments. Doing this enhances collaboration and provides high availability.
+.Prerequiste
+* A Git repository on an NFS server is available.
+
+.Procedure
+. Set up a {EAP} 7.1 cluster as described in  https://access.redhat.com/documentation/en-us/reference_architectures/2017/html-single/configuring_a_red_hat_jboss_eap_7_cluster/[_Configuring a {EAP} 7 Cluster_].
+. Install and configure Elasticsearch as described in <<clustering-elasticsearch-proc_{context}>>.
+. Install and configure AMQ Broker as described in <<clustering-artemis-activate-proc_{context}>>.
+. Install and configure {CENTRAL} as described in <<clustering-dm-install-proc_{context}>>.
+
+== Clustering {KIE_SERVER} in a runtime environment
+In a runtime environment, {KIE_SERVER} runs rules and processes that support business decisions. The primary benefit of clustering a {KIE_SERVER} runtime environment is load balancing. If activity on one node of the cluster increases, that activity can be shared among the other nodes of the cluster to improve performance.
+
+=== Clustering {KIE_SERVER} in a standalone runtime environment
 
-For example, if clustering is configured and Developer X is authoring rules on {CENTRAL} Node1  then Developer Y, who is working on {CENTRAL} Node2, can see the rules authored by Developer X as they are created. Furthermore, all of the developers working on individual nodes of the cluster will see the same synchronized content.
+.Prerequiste
+* A Git repository on an NFS server is available.
 
-In addition, clustering provides high availability in the rules development environment. If Developer X is working on Node1 and that node fails, Developer X's work is preserved and visible on any other node of the cluster.
+.Procedure
+. Set up a {EAP} 7.1 cluster as described in  https://access.redhat.com/documentation/en-us/reference_architectures/2017/html-single/configuring_a_red_hat_jboss_eap_7_cluster/[_Configuring a {EAP} 7 Cluster_].
+. Install and configure Elasticsearch as described in <<clustering-elasticsearch-proc_{context}>>.
+. Install and configure AMQ Broker as described in <<clustering-artemis-activate-proc_{context}>>.
```

Comment:
IIRC, The process server doesn't require AMQ, it works with any JMS provider. @mswiderski?